spec/carrierwave-aws_spec.rb

```diff
@@ -14,6 +14,19 @@
   end
 
   describe '#aws_acl' do
+    let(:derived_uploader) { FeatureUploader }
+
+    before do
+      # Reset uploader classes
```

Comment:
We should probably [reset_config](override https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/uploader/configuration.rb#L155) and include the `aws_` configurations. Then `uploader.reset_config` can be called here instead.